css/index.css

```diff
@@ -1 +1,306 @@
-/* Use your CSS from RWD - I here.  IF you don't have CSS, use the solution CSS from RWD - I*/
\ No newline at end of file
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Default styles
+============================ */
+
+:root {
+
+    /* Site colors */
+    --dark: #111;
+    --light: #fff;
+    --darkAccent: grey;
+    --lightAccent: lightgrey;
+
+    /* Site fonts */
+    --primaryFont: 'Titillium Web', sans-serif;
+    --secondaryFont: 'Bangers', sans-serif;
+
+    /* Alternate Site Fonts */
+    /* --primaryFont: 'Karla', sans-serif; */
+    /* --secondaryFont: 'Niconne', cursive; */
+}
+
+* {
+    box-sizing: border-box;
+}
+
+html {
+    font-size: 62.5%;
+}
+
+body {
+    font-family: var(--primaryFont);
+}
+
+h1 {
+    font-size: 2.4rem;
+}
+
+h3 {
+    font-size: 2.2rem;
+}
+
+p, a, address {
+    font-size: 1.6rem;
+}
+
+.cotainer {
+    width: 95%;
+}
+
+.horizontal-line {
+    color: var(--dark);
+}
+
+/* Custom Styles
+============================ */
+
+.logo {
+    display: block;
+    margin: 0 auto;
+    padding: 1.5rem 0 0 0;
+}
+
+/* Navigation
+============================ */
+
+nav {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 2rem;
+}
+
+.nav-link {
+    width: 100%;
+    text-decoration: none;
+    border-top: 0.1rem solid var(--lightAccent);
```

Comment:
I don't think I've seen rems used for border thickness, but it seems perfectly reasonable to me.